Worker/Worker.cs

```diff
@@ -22,6 +23,12 @@ public static void Main()
 
             Log.Logger.Information("Actor system joined cluster");
 
+            // allow process to exit when Control + C is invoked
+            Console.CancelKeyPress += (sender, e) =>
+            {
+                CoordinatedShutdown.Get(actorsytem).Run(CoordinatedShutdown.ClrExitReason.Instance).Wait();

```

Comment:
Need to `Wait` on the `Task` returned by `CoordinatedShutdown.Run()` during this method in order to ensure that the graceful cluster-exit code gets a chance to complete before the process is terminated.

This allows for nodes to leave the cluster cleanly, resulting in this output when we `Control + C` one of the worker processes:

![image](https://user-images.githubusercontent.com/326939/49256381-7678d400-f3f4-11e8-96d8-644f6ac08f7d.png)